app/javascript/src/components/Dashboard/Notes/HeaderMenu.jsx

```diff
@@ -0,0 +1,75 @@
+// @ts-nocheck
+import React, { useState } from "react";
+
+// @ts-ignore
+import { Search, Settings, Plus } from "@bigbinary/neeto-icons";
+import { Typography } from "@bigbinary/neetoui";
+import { MenuBar } from "@bigbinary/neetoui/layouts";
+// @ts-ignore
+// @ts-ignore
+
+const HeaderMenu = ({ showMenu }) => {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <div className="flex">
+      <MenuBar showMenu={showMenu} title="Contacts">
+        <MenuBar.Block active count={200} label="All" />
+        <MenuBar.Block count={80} label="Users" />
+        <MenuBar.Block count={60} label="Leads" />
+        <MenuBar.Block count={60} label="Visitors" />
+        <MenuBar.SubTitle
+          iconProps={[
+            {
+              icon: Search,
+              onClick: () => setIsSearchCollapsed(!isSearchCollapsed),
```

Comment:
Same as prev comment.

Reply:
@aagneyanNamboodiri Sorry about that. Added the wrong comment. To fix the issue go through https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update

Reply:
>I didn't touch this part of the code as it was the from the[ neetoUI component that i used](https://neetoui.onrender.com/?path=/docs/layouts-menubar--menu-bar-story)

That's not a good way to deal with code. When you copy paste something you ought to verify what you are pasting. Blind assumptions like things would always work or is perfect is not a good way mentality.

This also brings the question whether you had either gone through https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update or have understood that section very well. Because if you had understood that section very well and if you had gone through the copy pasted code, then as part of your self-review you should've fixed the issue. I am assuming that you didn't  verify the copy pasted code and thus the issue with the code.

Either ways, there's already an issue for this: https://github.com/bigbinary/neeto-ui/issues/1315

Reply:
I had not gone through the verified copy pasted code. 
Moving forward, I will take note of this.